src/assets/CollectiblesController.ts

```diff
@@ -1045,14 +1044,25 @@ export class CollectiblesController extends BaseController<
    *
    * @param collectible - The collectible object to check and update.
    * @param batch - A boolean indicating whether this method is being called as part of a batch or single update.
+   * @param accountParams - (optional) - The userAddress and chainId to check ownership against
```

Comment:
I know we designate optional arguments by wrapping them in square brackets when we write JavaScript, but I think the same rule applies in TypeScript, no?

Reply:
In typescript I actually think adding indications that its an optional param is redundant because this is indicated in the typing. Removing this altogether: https://github.com/MetaMask/controllers/pull/672/commits/0dcc89542336640a1f8dccb2cd9bca82b4c1dd03